src/screens/SendSheet.js

```diff
@@ -282,9 +286,15 @@ export default function SendSheet(props) {
 
   useEffect(() => {
     const updateNetworkAndProvider = async () => {
+      const assetNetwork =
+        selected?.type === AssetType.token || selected?.type === AssetType.nft
+          ? 'mainnet'
```

Comment:
Here it should be network from accountSettings, otherwise we'd be breaking all tesnets